blocks/bricks/recurrent.py

```diff
@@ -630,3 +631,231 @@ def apply(self, *args, **kwargs):
     @apply.delegate
     def apply_delegate(self):
         return self.children[0].apply
+
+
+class RecurrentStack(BaseRecurrent, Initializable):
+    u"""Stack of recurrent networks.
+
+    Build a stack of recurrent layers from a supplied list of
+    BaseRecurrent objects. Each object must have a `sequences`,
+    `contexts`, `states` and `outputs` parameters to its `apply` method.
```

Comment:
Can you please have a look at the code for recurrent decorator? Perhaps it has an error in its docstring saying inputs and not sequences  and if so this should be fixed. However, if you look at the code when handling return_initial_states case there is a reference to .owner.inputs (I guess this is OK because return_initial_states is in use in SequenceGenerator)